demo/helper-scripts/demo-pod-with-discovery.json.parsec.template

```diff
@@ -0,0 +1,25 @@
+{
+    "apiVersion": "v1",
+    "kind": "Pod",
+    "metadata": {
+        "name": "demo-parsec-NUM"
+    },
+    "spec": {
+        "containers": [
+            {
+                "image": "intelsdi/node-feature-discovery-APP",
```

Comment:
It would be nice to also include the image sources (e.g. as a git-submodule) so anyone can rebuild /publish them and it's not just a random container from the internet.